lighthouse-core/config/default-config.js

```diff
@@ -31,6 +31,38 @@ const UIStrings = {
   diagnosticsGroupTitle: 'Diagnostics',
   /** Description of the diagnostics section of the Performance category. Within this section are audits with non-imperative titles that provide more detail on the page's page load performance characteristics. Whereas the 'Opportunities' suggest an action along with expected time savings, diagnostics do not. Within this section, the user may read the details and deduce additional actions they could take. */
   diagnosticsGroupDescription: 'More information about the performance of your application.',
+  /* Title of the color contrast section within the Accessibility category. Within this section are audits with descriptive titles that highlight the color and vision aspects of the page's accessibility that are passing or failing. */
+  a11yColorContrastGroupTitle: 'Color Contrast Is Satisfactory',
```

Comment:
'Color Contrast ~~Is~~ is Satisfactory'?

This happens a few times, do we mean to capitalize these words every time, or do we need to look at them to make sure they are following a capitalization guideline?

Reply:
I've got no dog in this fight just moved the string up :)

I think the thinking here was that we have very few such small non-title-cased words that it just stood out too much, but 🤷‍♂️